bin/evaluate

```diff
@@ -13,7 +13,8 @@ import argparse
 import warnings
 
 from madmom.utils import search_files, match_file
-from madmom.evaluation import alignment, chords, beats, notes, onsets, tempo
+from madmom.evaluation import (alignment, chords, beats, key, notes, onsets,
+                               tempo)
```

Comment:
Since the line is changes anyways, you could reorder the imports alphabetically.